### PR TITLE
Add CLIQZ 1.0.2

### PR DESCRIPTION
**Pull request by kassi**

### CODE REVIEW

File: Casks/cliqz.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask :v1 => 'cliqz' do
```
Comment (vitorgalvao) [inline Casks/cliqz.rb]: `cask 'cliqz' do`

File: Casks/cliqz.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  version '1.0.2'
+  sha256 '9369fefb65b36a9ac465b2d0096de61822d40b8168ddfa8a0d6e31652cefe2e6'
+
+  url "http://repository.cliqz.com.s3.amazonaws.com/dist/release/#{version}/de/CLIQZ-#{version}.de.mac.dmg"
```
Comment (vitorgalvao) [inline Casks/cliqz.rb]: Missing [`name`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/name.md).

Comment (vitorgalvao) [inline Casks/cliqz.rb]: Needs a `# repository.cliqz.com.s3.amazonaws.com was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

File: Casks/cliqz.rb (hunk 1 above)
Comment (kassi) [inline Casks/cliqz.rb]: I took a look into some others as examples and neither one uses the form you describe. All use `cask :v1 => 'name' do`

Comment (kassi) [inline Casks/cliqz.rb]: Ok, looks like I was working on an outdated version. Sorry. Will update soon.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as aa2eceba91fbabcdb537bf1e820c567cd71170d6 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**